newdle/client/src/components/Navigator.js

```diff
@@ -47,7 +47,7 @@ Step.propTypes = {
 function StepBody({title, description}) {
   return (
     <div className={styles['step-body']}>
-      <Header className={styles['title']}>{title}</Header>
+      <h3>{title}</h3>
```

Comment:
I am wondering whether it would look nicer with the text & description texts being centered

Reply:
Like this?

![image](https://user-images.githubusercontent.com/2699/65407307-69ddc600-dde2-11e9-9406-8d38ce638658.png)

Reply:
Hm, TBH this one doesn't look that nice, however maybe we could decrease the width of this element and make it more in line with the `Header` on the splash page.

![splash](https://user-images.githubusercontent.com/3616940/65407502-056f3680-dde3-11e9-8cc0-e8d83ce9354d.png)

Reply:
I guess we can discuss this afterwards, it shouldn't affect this PR.

Reply:
But very good point, it would make the design nicer.